contracts/LifCrowdsale.sol

```diff
@@ -131,17 +150,23 @@ contract LifCrowdsale is Ownable, Pausable {
     // calculate token amount to be created
     uint256 tokens = weiAmount.mul(rate);
 
-    // update state
-    weiRaised = weiRaised.add(weiAmount);
-    purchases[beneficiary] = weiAmount;
-    tokensSold = tokensSold.add(tokens);
+    // check if we are in the public presale block range
+    if (rate == publicPresaleRate) {
```

Comment:
I think its better this way because they can use the fallback function at any time during the public presale and ICO, will be easier for investors to send the funds. Also more secure, if we have to publish instructions about how to invest on the public presale building a function will be more difficult for the investor and he can make a mistake. Using fallback for me is the way to go. If we decide to change it we can do it on another PR.